src/software/SfMWebGLViewer/config.h.in

```diff
@@ -0,0 +1,6 @@
+#ifndef _OPENMVG_MAIN_OPENMVG_TO_WEBGL_CONFIG_H_
+#define _OPENMVG_MAIN_OPENMVG_TO_WEBGL_CONFIG_H_
+
+#define OPENMVG_SFM_WEBGL_PATH "@OPENMVG_SOFTWARE_SFMWEBGLVIEWER_SRC_DIR@"
```

Comment:
This path will not be valid when an install is performed.
So I think it would be better to always ask the resources directory in the binary openMVG_main_openMVG2WebGL